ArduPlane/ArduPlane.cpp

```diff
@@ -963,5 +966,18 @@ void Plane::publish_osd_info()
     osd.set_nav_info(nav_info);
 }
 #endif
-
+#if FRSKY_TELEM_ENABLED == ENABLED
+void Plane::update_nav_info()
+{
+    AP_Frsky_Telem::NavInfo nav_info;
+    //
+    nav_info.wp_distance = auto_state.wp_distance;
```

Comment:
This isn't like all the others if I'm thinking right.  Some combination of guided_WP_loc and next_WP_loc would be more analogous

Reply:
Hi, I'm still learning my way through vehicle code, I "borrowed" this from the recent OSD code 
https://github.com/ArduPilot/ardupilot/blob/c0af0cfe70a8a102b1d7dfa436a46d7c32039ce3/ArduPlane/ArduPlane.cpp#L967-L979
and I must admit I simply I assumed it was all right. having said that it really is OK with me to improve on the quality of what I proposed!

Reply:
Similarly, auto_state.wp_distance is what is used by Plane::send_nav_controller_output() to send nav info via MAVLink...

We may also need to consider the case of quadplanes in vtol: https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/GCS_Mavlink.cpp#L181